rllib/offline/estimators/tests/test_feature_importance.py

```diff
@@ -8,7 +8,7 @@
 
 class TestFeatureImportance(unittest.TestCase):
     def setUp(self):
-        ray.init()
+        ray.init(num_cpus=4)
```

Comment:
No need to specify CPUs. In fact, don't. because we may run these test on somewhere that does not have this many resources.